features/cli.feature

```diff
@@ -35,6 +35,9 @@ Feature: CLI
       """
     And a git_base option appended to "modulesync.yml" for local tests
     And a directory named "moduleroot"
-    When I run `msync update --noop --namespace fakenamespace`
+    When I run `msync update --noop --namespace fakenamespace --branch command-line-branch`
```

Comment:
Kind of feels like an unrelated change. Why is it needed here?

Reply:
This test checks that `modulesync` CLI arguments supersede the options specified in configuration file. The way to do this is to check the CLI output... but the output changes (like in this commit), we need to change this test and can be disturbing.

So, to match with the goal of this test, I change it by now checking the branch as the text `Syncing xxx/yyy` now uses the `given_name` instead of `#{namespace}/#{name}`.

Reply:
Personally I would submit it as a separate PR. Not going to push for it now and I'm ok merging this, but small focused PRs tend to be reviewed & merged quicker.

Reply:
I agree.